views/js/admin/tabsHandlerBelowPs174.js

```diff
@@ -0,0 +1,39 @@
+
+$(document).ready(function (){
+    var tabs = $('.page-head-tabs > a');
+    var visible = visibleTabs;
+    var seen = {};
+    $( tabs ).each(function() {
+        var txt = $(this).text();
+        if (seen[txt]) {
+            $(this).remove();
+        } else {
+            seen[txt] = true;
+        }
+
+        var href = $( this ).attr('href');
+        var controller = getUrlParam('controller', href);
+        if (!inArray(controller, visible)) {
+           $(this).remove();
+        }
+    });
+
+    function getUrlParam( name, url ) {
+        if (!url) url = location.href;
+        name = name.replace(/[\[]/,"\\\[").replace(/[\]]/,"\\\]");
```

Comment:
add function or const where you could explain what it does with name

Reply:
Truly, no clue, but it is not needed -.-